<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaAssemblerClass</class>
 <widget class="QMainWindow" name="MetaAssemblerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>526</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MetaAssembler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTextEdit" name="textEdit"/>
       <widget class="QTableView" name="tableView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="statusText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>170</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionAssemble_to_CDM"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionClear_Status_View"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionEditor"/>
    <addaction name="actionCDM_Table_View"/>
    <addaction name="actionStatus_View"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCustomise"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action68K_General_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolbarNew"/>
   <addaction name="actionToolbarOpen"/>
   <addaction name="actionToolbarSave"/>
   <addaction name="separator"/>
   <addaction name="actionToolbarCut"/>
   <addaction name="actionToolbarCopy"/>
   <addaction name="actionToolbarPaste"/>
   <addaction name="separator"/>
   <addaction name="actionConvertToCDM"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionDocument">
   <property name="text">
    <string>Document</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionCDM_Table_View">
   <property name="text">
    <string>CDM Table View</string>
   </property>
  </action>
  <action name="actionCustomise">
   <property name="text">
    <string>Customise</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="action68K_General_Help">
   <property name="text">
    <string>68K General Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionToolbarNew">
   <property name="icon">
    <iconset>
     <normaloff>Resources/new.png</normaloff>Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionToolbarOpen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionToolbarSave">
   <property name="icon">
    <iconset>
     <normaloff>Resources/save.png</normaloff>Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionToolbarReformat">
   <property name="icon">
    <iconset>
     <normaloff>Resources/refresh.png</normaloff>Resources/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reformat</string>
   </property>
  </action>
  <action name="actionToolbarCut">
   <property name="icon">
    <iconset>
     <normaloff>Resources/cut.png</normaloff>Resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionToolbarCopy">
   <property name="icon">
    <iconset>
     <normaloff>Resources/copy.png</normaloff>Resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionToolbarPaste">
   <property name="icon">
    <iconset>
     <normaloff>Resources/paste.png</normaloff>Resources/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionError_Check">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/png/24x24/Danger.png</normaloff>../../../../Desktop/png/24x24/Danger.png</iconset>
   </property>
   <property name="text">
    <string>Error Check</string>
   </property>
  </action>
  <action name="actionConvertToCDM">
   <property name="icon">
    <iconset>
     <normaloff>Resources/component.png</normaloff>Resources/component.png</iconset>
   </property>
   <property name="text">
    <string>ConvertToCDM</string>
   </property>
  </action>
  <action name="actionError_Check_2">
   <property name="text">
    <string>Error Check</string>
   </property>
  </action>
  <action name="actionAssemble_to_CDM">
   <property name="text">
    <string>Assemble to CDM</string>
   </property>
  </action>
  <action name="actionStatus_View">
   <property name="text">
    <string>Status View</string>
   </property>
  </action>
  <action name="actionClear_Status_View">
   <property name="text">
    <string>Clear Status View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MetaAssembler.qrc"/>
 </resources>
 <connections/>
</ui>
